tests/run.sh

```diff
@@ -29,6 +29,13 @@ if [ "${1-}" = '--debug' ]; then
 fi
 
 for script in tests/${TEST_NAME-*}/run.sh; do
+    TEST_NAME="$(basename "$(dirname "$script")")"
+    if [ $TEST_NAME = "br_tls" ]; then
+        echo "FIXME enable br_tls tests"
+        echo "TiKV master (ed71f20f445e10595553d2bf3d1a1eb645b9a61a) aborts when TLS is enabled"
+        continue
+    fi
```

Comment:
is this PR blocked by tikv/tikv#7209?

Reply:
Yes, tikv core dumps 100% and fails BR CI.